lib/grppc.gi

```diff
@@ -1852,6 +1852,87 @@ InstallGlobalFunction( GapInputPcGroup, function(U,name)
 
 end );
 
+#############################################################################
+##
+#F  PrintPcPresentation( <grp>, <commBool> )  . . . . . . .  gap input string
+##
+##  Display the pc relations of a pc group.
+##  If commBool the commutator presentation is printed,
+##  otherwise the power presentation.
+##  Trivial commutators / powers are not printed.
+##  The generators  will  be  named "g<i>".
+##
+InstallGlobalFunction( PrintPcPresentation, function(G, commBool)
+    local pcgs, n, F, gens, i, pis, exp, t, h, rel, commPower, j;
+
+    pcgs:=Pcgs(G);
+    n:=Length(pcgs);
+    if n=0 then
+        Print("Trivial Group\n");
+        return;
+    fi;
+    F    := FreeGroup( n, "g" );
+    gens := GeneratorsOfGroup( F );
+    pis  := RelativeOrders( pcgs );
+
+    # the power
+    for i in [1..n] do
+        exp := ExponentsOfRelativePower( pcgs, i ){[i+1..n]};
+        t   := One( F );
+        for h in [i+1..n] do
+            t := t * gens[h]^exp[h-i];
+        od;
```

Comment:
I think (but have not verified) that the previous four lines can be replaced by just this: `t := PcElementByExponentsNC( pcgs, exp );`. Similar elsewhere in this function.

Reply:
I simply tested this and its not correct for e.g. C10.
From the test output:

> ########> Diff in tst/testinstall/grppc.tst:5
> # Input is:
> Display(CyclicGroup(IsPcGroup, 10));
> # Expected output:
> pc-group with relations:
> g1^2 = g2
> g2^5 = id
> All other pairs of generators commute.
> # But found:
> pc-group with relations:
> g1^2 = f1
> g2^5 = id
> g2 ^ g1 = f1
> ########

Reply:
That comment of mine was simply nonsense, sorry.